_posts/en/newsletters/2022-04-27-newsletter.md

```diff
@@ -0,0 +1,240 @@
+---
+title: 'Bitcoin Optech Newsletter #197'
+permalink: /en/newsletters/2022/04/27/
+name: 2022-04-27-newsletter
+slug: 2022-04-27-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## News
+
+- **Discussion about activating CTV:** Jeremy Rubin [posted][rubin
+  ctv-st] to the Bitcoin-Dev mailing list his plan to release software
+  that will allow miners to begin signaling whether they intend to
+  enforce the [BIP119][] rules for the proposed
+  [OP_CHECKTEMPLATEVERIFY][topic op_checktemplateverify] (CTV) opcode.
```

Comment:
Worth noting maybe that Jeremy has indicated (in a public Telegram channel and elsewhere) that he plans to amend the ST parameters that he initially mentioned in the blog post:
- https://twitter.com/JeremyRubin/status/1518477022439247872
- https://t.me/op_ctv/15562

Reply:
I know you're not Jeremy's minder James but as someone who seems to support an upcoming CTV soft fork activation attempt and presumably a participant in the CTV discussions I'm staggered you are comfortable with activation params being dictated on a whim by Jeremy via Twitter and messaging apps. I think even if I was supportive of a soft fork and it had community consensus that would set off alarm bells for me. Regardless I agree with @harding below that the mailing list should be informed. 

Ideally Optech would have covered discussions on the mailing list in previous months about the dangers of attempting soft forks without community consensus but for some reason it has totally ignored them. I'm not quite sure why that was. Presumably for the same reasons why it ignored the UASF client for Taproot and it is ignoring the URSF discussions today. Frankly I think it is irresponsible. But I know it won't make any difference so I won't comment further.

Reply:
@michaelfolkson 

> Ideally Optech would have covered discussions on the mailing list in previous months about the dangers of attempting soft forks without community consensus but for some reason it has totally ignored them.

We haven't ignored them: https://bitcoinops.org/en/newsletters/2022/03/23/#speedy-trial-discussion

> I won't comment further.

Thanks, I appreciate that.